src/components/Labs/getLabs.tsx

```diff
@@ -0,0 +1,29 @@
+export function getLabs() {
```

Comment:
```suggestion
export const laboratories: Laboratory[]= []
```

Добавить папку ./mocks  в ./src в нее туда закинь этот файл и назови laboratories.ts

Reply:
done